app/Resources/assets/js/lsdoc/index.js

```diff
@@ -649,3 +649,10 @@ $(window).resize(function(){
     $('#treeSideLeft').height($("#treeView").height());
 });
 
+
+$(".export_csv_comment").click(function(){
+    var currentUrl=window.location.href;
+    var urlArray = currentUrl.split('/');
+    var url="/salt/case/export_comment/"+urlArray[urlArray.length-2]+"/"+urlArray[urlArray.length-1]+"/comment.csv";
```

Comment:
You should be able to get the item id from the view by exposing it to a JavaScript variable, you don't need to ask the URL directly. Getting it from the URL itself could lead to troubles in the future if the URL format changes.

Reply:
Yes, We might be get item id by you're suggested way. We discussed this thing in one of hotline and we were not sure how to get modified id by that way. Even you were fine with current used way that time. If you still think we should need to use that way only, then I need some time to research on it.

Reply:
We need to refactor this. As I said in the hotline the controller is returning the `itemId` and the `documentId` so you should be able to get this values in the view.
Another way is getting the id from the node in the same way the comment is created, please check `comment.js` file to see how the comment is created.